EKF/EKFGSF_yaw.h

```diff
@@ -98,18 +98,18 @@ class EKFGSF_yaw
 	// Declarations used by a bank of N_MODELS_EKFGSF EKFs
 
 	struct _ekf_gsf_struct{
-		matrix::Vector3f X; // Vel North (m/s),  Vel East (m/s), yaw (rad)s
-		matrix::SquareMatrix<float, 3> P; // covariance matrix
-		matrix::SquareMatrix<float, 2> S_inverse;  // inverse of the innovation covariance matrix
-		float S_det_inverse; // inverse of the innovation covariance matrix determinant
-		matrix::Vector2f innov; // Velocity N,E innovation (m/s)
-	};
-	_ekf_gsf_struct _ekf_gsf[N_MODELS_EKFGSF];
-	bool _vel_data_updated;  // true when velocity data has been updated
-	bool _run_ekf_gsf;       // true when operating condition is suitable for to run the GSF and EKF models and fuse velocity data
-	Vector2f _vel_NE;        // NE velocity observations (m/s)
-	float _vel_accuracy;     // 1-sigma accuracy of velocity observations (m/s)
-	bool _ekf_gsf_vel_fuse_started;	// true when the EKF's have started fusing velocity data and the prediction and update processing is active
+		matrix::Vector3f X; 				// Vel North (m/s),  Vel East (m/s), yaw (rad)s
```

Comment:
FYI it should probably be tabs for indentation spaces for alignment.

Reply:
Thanks, I'll keep that in mind for next time